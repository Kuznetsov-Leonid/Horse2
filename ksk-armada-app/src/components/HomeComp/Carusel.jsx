import React from 'react';

const Carusel = () => {
    return(
        <>
    
        </>
    );
}

export default Carusel