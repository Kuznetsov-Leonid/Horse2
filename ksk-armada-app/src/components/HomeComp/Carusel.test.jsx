import { render, screen } from '@testing-library/react';
import Carusel from './Carusel';

describe('Carusel component', () => {
    render(<Carusel />);
    expect(screen.queryByRole('Carusel')).toBeNull();
});

describe('Carusel snapshot', () => {
    it('Carusel snapshot', () => {
        const CaruselSnap = render(<Carusel />);
        expect(CaruselSnap).toMatchSnapshot();
    });
});