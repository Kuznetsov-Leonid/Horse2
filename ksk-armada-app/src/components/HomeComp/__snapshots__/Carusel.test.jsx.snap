// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carusel snapshot Carusel snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="carousel slide carousel-fade"
      >
        <div
          class="carousel-indicators"
        >
          <button
            aria-current="true"
            aria-label="Slide 1"
            class="active"
            data-bs-target=""
            type="button"
          />
          <button
            aria-current="false"
            aria-label="Slide 2"
            data-bs-target=""
            type="button"
          />
          <button
            aria-current="false"
            aria-label="Slide 3"
            data-bs-target=""
            type="button"
          />
          <button
            aria-current="false"
            aria-label="Slide 4"
            data-bs-target=""
            type="button"
          />
          <button
            aria-current="false"
            aria-label="Slide 5"
            data-bs-target=""
            type="button"
          />
        </div>
        <div
          class="carousel-inner"
        >
          <div
            class="active carousel-item"
          >
            <div
              class="sc-bczRLJ bIDtHO"
            >
              <div
                class="JumboPrise condComp"
              >
                <div
                  class="container"
                >
                  <div
                    class="overlay card"
                  >
                    <div
                      class="row"
                    >
                      <h1>
                        Спортивные мероприятия
                      </h1>
                      <div
                        class="col"
                      >
                        <h6>
                          На нашей базе регулярно проходят соревнования по конкуру и конной выездке.
                        </h6>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="carousel-item"
          >
            <div
              class="sc-bczRLJ bIDtHO"
            >
              <div
                class="JumboPrise condComp"
              >
                <div
                  class="container"
                >
                  <div
                    class="overlay card"
                  >
                    <div
                      class="row"
                    >
                      <h1>
                        Обучение верховой езде
                      </h1>
                      <div
                        class="col"
                      >
                        <h6>
                          Обучаем с основ до профессионального уровня, даже если Вы до этого никогда не были в седле. Уроки проходят в группах или индивидуально, а лошадки подбираются в соответствии с возрастом, весом и навыками всадника.
                        </h6>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="carousel-item"
          >
            <div
              class="sc-bczRLJ bIDtHO"
            >
              <div
                class="JumboPrise condComp"
              >
                <div
                  class="container"
                >
                  <div
                    class="overlay card"
                  >
                    <div
                      class="row"
                    >
                      <h1>
                        Конные прогулки
                      </h1>
                      <div
                        class="col"
                      >
                        <h6>
                          Наслаждайтесь природой и общением с животными. Лучший выбор для семейного отдыха и встреч с друзьями.
                        </h6>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="carousel-item"
          >
            <div
              class="sc-bczRLJ bIDtHO"
            >
              <div
                class="JumboPrise condComp"
              >
                <div
                  class="container"
                >
                  <div
                    class="overlay card"
                  >
                    <div
                      class="row"
                    >
                      <h1>
                        Постой лошадей
                      </h1>
                      <div
                        class="col"
                      >
                        <h6>
                          Создавать комфортные условия для жизни лошадей — наша работа!
                        </h6>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="carousel-item"
          >
            <div
              class="sc-bczRLJ bIDtHO"
            >
              <div
                class="JumboPrise condComp"
              >
                <div
                  class="container"
                >
                  <div
                    class="overlay card"
                  >
                    <div
                      class="row"
                    >
                      <h1>
                        Фотосессии с лошадьми
                      </h1>
                      <div
                        class="col"
                      >
                        <h6>
                          Чтoбы вaши впeчaтлeния сoхрaнились нe тoлькo в вaшeй пaмяти, мы прeдлaгaeм прoфeссиoнaльную фoтoсeссию с лoшaдьми.
                        </h6>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <a
          class="carousel-control-prev"
          href="#"
          role="button"
          tabindex="0"
        >
          <span
            aria-hidden="true"
            class="carousel-control-prev-icon"
          />
          <span
            class="visually-hidden"
          >
            Previous
          </span>
        </a>
        <a
          class="carousel-control-next"
          href="#"
          role="button"
          tabindex="0"
        >
          <span
            aria-hidden="true"
            class="carousel-control-next-icon"
          />
          <span
            class="visually-hidden"
          >
            Next
          </span>
        </a>
      </div>
    </div>
    <div>
      <div
        class="carousel slide carousel-fade"
      >
        <div
          class="carousel-indicators"
        >
          <button
            aria-current="true"
            aria-label="Slide 1"
            class="active"
            data-bs-target=""
            type="button"
          />
          <button
            aria-current="false"
            aria-label="Slide 2"
            data-bs-target=""
            type="button"
          />
          <button
            aria-current="false"
            aria-label="Slide 3"
            data-bs-target=""
            type="button"
          />
          <button
            aria-current="false"
            aria-label="Slide 4"
            data-bs-target=""
            type="button"
          />
          <button
            aria-current="false"
            aria-label="Slide 5"
            data-bs-target=""
            type="button"
          />
        </div>
        <div
          class="carousel-inner"
        >
          <div
            class="active carousel-item"
          >
            <div
              class="sc-bczRLJ bIDtHO"
            >
              <div
                class="JumboPrise condComp"
              >
                <div
                  class="container"
                >
                  <div
                    class="overlay card"
                  >
                    <div
                      class="row"
                    >
                      <h1>
                        Спортивные мероприятия
                      </h1>
                      <div
                        class="col"
                      >
                        <h6>
                          На нашей базе регулярно проходят соревнования по конкуру и конной выездке.
                        </h6>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="carousel-item"
          >
            <div
              class="sc-bczRLJ bIDtHO"
            >
              <div
                class="JumboPrise condComp"
              >
                <div
                  class="container"
                >
                  <div
                    class="overlay card"
                  >
                    <div
                      class="row"
                    >
                      <h1>
                        Обучение верховой езде
                      </h1>
                      <div
                        class="col"
                      >
                        <h6>
                          Обучаем с основ до профессионального уровня, даже если Вы до этого никогда не были в седле. Уроки проходят в группах или индивидуально, а лошадки подбираются в соответствии с возрастом, весом и навыками всадника.
                        </h6>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="carousel-item"
          >
            <div
              class="sc-bczRLJ bIDtHO"
            >
              <div
                class="JumboPrise condComp"
              >
                <div
                  class="container"
                >
                  <div
                    class="overlay card"
                  >
                    <div
                      class="row"
                    >
                      <h1>
                        Конные прогулки
                      </h1>
                      <div
                        class="col"
                      >
                        <h6>
                          Наслаждайтесь природой и общением с животными. Лучший выбор для семейного отдыха и встреч с друзьями.
                        </h6>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="carousel-item"
          >
            <div
              class="sc-bczRLJ bIDtHO"
            >
              <div
                class="JumboPrise condComp"
              >
                <div
                  class="container"
                >
                  <div
                    class="overlay card"
                  >
                    <div
                      class="row"
                    >
                      <h1>
                        Постой лошадей
                      </h1>
                      <div
                        class="col"
                      >
                        <h6>
                          Создавать комфортные условия для жизни лошадей — наша работа!
                        </h6>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="carousel-item"
          >
            <div
              class="sc-bczRLJ bIDtHO"
            >
              <div
                class="JumboPrise condComp"
              >
                <div
                  class="container"
                >
                  <div
                    class="overlay card"
                  >
                    <div
                      class="row"
                    >
                      <h1>
                        Фотосессии с лошадьми
                      </h1>
                      <div
                        class="col"
                      >
                        <h6>
                          Чтoбы вaши впeчaтлeния сoхрaнились нe тoлькo в вaшeй пaмяти, мы прeдлaгaeм прoфeссиoнaльную фoтoсeссию с лoшaдьми.
                        </h6>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <a
          class="carousel-control-prev"
          href="#"
          role="button"
          tabindex="0"
        >
          <span
            aria-hidden="true"
            class="carousel-control-prev-icon"
          />
          <span
            class="visually-hidden"
          >
            Previous
          </span>
        </a>
        <a
          class="carousel-control-next"
          href="#"
          role="button"
          tabindex="0"
        >
          <span
            aria-hidden="true"
            class="carousel-control-next-icon"
          />
          <span
            class="visually-hidden"
          >
            Next
          </span>
        </a>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="carousel slide carousel-fade"
    >
      <div
        class="carousel-indicators"
      >
        <button
          aria-current="true"
          aria-label="Slide 1"
          class="active"
          data-bs-target=""
          type="button"
        />
        <button
          aria-current="false"
          aria-label="Slide 2"
          data-bs-target=""
          type="button"
        />
        <button
          aria-current="false"
          aria-label="Slide 3"
          data-bs-target=""
          type="button"
        />
        <button
          aria-current="false"
          aria-label="Slide 4"
          data-bs-target=""
          type="button"
        />
        <button
          aria-current="false"
          aria-label="Slide 5"
          data-bs-target=""
          type="button"
        />
      </div>
      <div
        class="carousel-inner"
      >
        <div
          class="active carousel-item"
        >
          <div
            class="sc-bczRLJ bIDtHO"
          >
            <div
              class="JumboPrise condComp"
            >
              <div
                class="container"
              >
                <div
                  class="overlay card"
                >
                  <div
                    class="row"
                  >
                    <h1>
                      Спортивные мероприятия
                    </h1>
                    <div
                      class="col"
                    >
                      <h6>
                        На нашей базе регулярно проходят соревнования по конкуру и конной выездке.
                      </h6>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="carousel-item"
        >
          <div
            class="sc-bczRLJ bIDtHO"
          >
            <div
              class="JumboPrise condComp"
            >
              <div
                class="container"
              >
                <div
                  class="overlay card"
                >
                  <div
                    class="row"
                  >
                    <h1>
                      Обучение верховой езде
                    </h1>
                    <div
                      class="col"
                    >
                      <h6>
                        Обучаем с основ до профессионального уровня, даже если Вы до этого никогда не были в седле. Уроки проходят в группах или индивидуально, а лошадки подбираются в соответствии с возрастом, весом и навыками всадника.
                      </h6>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="carousel-item"
        >
          <div
            class="sc-bczRLJ bIDtHO"
          >
            <div
              class="JumboPrise condComp"
            >
              <div
                class="container"
              >
                <div
                  class="overlay card"
                >
                  <div
                    class="row"
                  >
                    <h1>
                      Конные прогулки
                    </h1>
                    <div
                      class="col"
                    >
                      <h6>
                        Наслаждайтесь природой и общением с животными. Лучший выбор для семейного отдыха и встреч с друзьями.
                      </h6>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="carousel-item"
        >
          <div
            class="sc-bczRLJ bIDtHO"
          >
            <div
              class="JumboPrise condComp"
            >
              <div
                class="container"
              >
                <div
                  class="overlay card"
                >
                  <div
                    class="row"
                  >
                    <h1>
                      Постой лошадей
                    </h1>
                    <div
                      class="col"
                    >
                      <h6>
                        Создавать комфортные условия для жизни лошадей — наша работа!
                      </h6>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="carousel-item"
        >
          <div
            class="sc-bczRLJ bIDtHO"
          >
            <div
              class="JumboPrise condComp"
            >
              <div
                class="container"
              >
                <div
                  class="overlay card"
                >
                  <div
                    class="row"
                  >
                    <h1>
                      Фотосессии с лошадьми
                    </h1>
                    <div
                      class="col"
                    >
                      <h6>
                        Чтoбы вaши впeчaтлeния сoхрaнились нe тoлькo в вaшeй пaмяти, мы прeдлaгaeм прoфeссиoнaльную фoтoсeссию с лoшaдьми.
                      </h6>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <a
        class="carousel-control-prev"
        href="#"
        role="button"
        tabindex="0"
      >
        <span
          aria-hidden="true"
          class="carousel-control-prev-icon"
        />
        <span
          class="visually-hidden"
        >
          Previous
        </span>
      </a>
      <a
        class="carousel-control-next"
        href="#"
        role="button"
        tabindex="0"
      >
        <span
          aria-hidden="true"
          class="carousel-control-next-icon"
        />
        <span
          class="visually-hidden"
        >
          Next
        </span>
      </a>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
